website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +88,48 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($(window).width() <= 600) {
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
+    }
+    else {
+        $('table.responsive-table-xxs tbody tr:not(:hidden) td>div.header').css('display', '');
+        $('table.responsive-table-xxs tbody tr td:first-child').removeAttr('role').removeAttr('onclick');
+    }
+    if ($(window).width() <= 768) {
+        $('table.responsive-table-xs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
+    }
+    else {
+        $('table.responsive-table-xs tbody td>div.header').css('display', '');
+        $('table.responsive-table-xs tbody tr td:first-child').removeAttr('role').removeAttr('onclick');
+    }
+};
+
+$(window).load(function() {
+    checkWindowWidth();
+    var linkTable = $('#privateLinkTable')[0];
+    if (linkTable !== undefined) {
+        rt.responsiveTable(linkTable);
+    }
+});
+
+$(window).resize(function() {
+    checkWindowWidth();
+});
+
+window.toggleExpand = function(el) {
```

Comment:
You shouldn't put this in the global namespace. I think it's related to the way you are handling the toggleExpland functionality overall.